src/Lecture1.hs

```diff
@@ -108,7 +123,15 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+
+strSum :: [Char] -> Int
+strSum text = 
+ let 
+     text1 = words text
+     str2 = map read text1
+     sum1 = sum str2
+      in sum1
```

Comment:
That's absolutely correct implementation! 👏🏻 
If you want, you can avoid introducing local variables here and implement it a simple one-liner:

```suggestion
strSum text = sum (map read (words text))
```